### ISSUE TITLE
Rescripts test isnt using babel config

### ISSUE THREAD

**Issue by robcalcroft** [harrysolovay/rescripts#17]
When I run my testing suite with `rescripts test`, the tests don't seem to pickup on my `.babelrc` file:
```
{
  "presets": ["react-app"],
  "plugins": [
    [
      "@babel/plugin-proposal-decorators", {
        "legacy": true,
      }
    ],
    [
      "babel-plugin-named-asset-import",
      {
        "loaderMap": {
          "svg": {
            "ReactComponent": "@svgr/webpack?-prettier,-svgo![path]",
          },
        },
      },
    ],
  ],
}
```
As such I get this error: `SyntaxError: /path/to/src/stores/index.js: Support for the experimental syntax 'decorators-legacy' isn't currently enabled` because i'm using decorators, but the babel plugin should resolve this.

The UI works fine when I run `rescripts start`

`.rescriptsrc.js`:
```
module.exports = ['env'];
```

**Comment by harrysolovay** [comment]
Hi @robcalcroft –– I haven't yet implemented this for Jest (there's some difficulty in accepting different Babel config source formats). Check out issue #11 for a solution.

TLDR; I do plan on building this into `@rescripts/rescript-use-babel-config` in a non-breaking way. For the time being, you'll need to edit your `.rescriptsrc.js` (the following should work considering you're using a `.babelrc`):

```js
module.exports = [
  ['use-babel-config', '.babelrc'],
  {
    jest: config => {
      config.transform['^.+\\.(js|jsx|ts|tsx)$'] = 'babel-jest';
      return config
    }
  }
]
```

Please let me know if this helps :)

**Comment by robcalcroft** [comment]
This did the job thanks 😄

**Comment by dustinlacewell** [comment]
Hey I was just interested in whether this was ever fixed?
